CHANGELOG.md

```diff
@@ -6,8 +6,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased] - ReleaseDate
 ### Added
 ### Changed
-- Changed `readlink` and `readlinkat` to return `osString`
+- Changed `readlink` and `readlinkat` to return `OsString`
   ([#1109](https://github.com/nix-rust/nix/pull/1109))
+
```

Comment:
This example isn't going to mean much to a reader.  I'm thinking something in English, like "the `buffer` argument of `readlink` and `readlinkat` has been removed, and the return value is now an owned type.  Existing code can be updated by removing the buffer argument and removing any `clone` or similar operation on the output."